### ISSUE TITLE
Form Item's Help Text May Overlap Contained Input's Counter

### ISSUE THREAD

**Issue by francisagyapong-goa** [GovAlta/ui-components#885]
Given the right width of containers, the help text on a form item can overlap with the character counter on an input. 

This behaviour has been observed in both Brave (Version 1.45.69 Chromium: 106.0.5249.55) and Firefox (106.0b3), Windows 10.

Library: `@abgov/web-components@1.0.0-alpha.103`

![2022-09-23_09-27-form-item-input-charcount-helptext-overlap-wrapping](https://user-images.githubusercontent.com/94571832/192004552-0a97f9ba-28ed-4d0f-b595-94979052f1f2.png)

![2022-09-23_09-29-form-item-input-charcount-helptext-overlap-no-wrap](https://user-images.githubusercontent.com/94571832/192004489-e7355690-bd54-4df5-9890-d6791b328122.png)

**Comment by ArakTaiRoth** [comment]
@francisagyapong-goa Can you supply the specific code that's causing the problem?

**Comment by ArakTaiRoth** [comment]
[DDIDS-1012]

**Comment by francisagyapong-goa** [comment]
> @francisagyapong-goa Can you supply the specific code that's causing the problem?

@ArakTaiRoth the code below should trigger it (you may need to slightly change the width of the wrapping div):

```html
<div style="margin: auto; padding: 1rem; border: 1px solid #dcdcdc; width: 400px">
  <goa-form-item label="This is a test" helptext="This information will be made available to Interested Suppliers as an opportunity update notification and in  the public Opportunity posting.">
    <goa-textarea width="100%" rows="4" maxcharcount="1000" showcounter></goa-textarea>
  </goa-form-item>
</div>
```

**Comment by ArakTaiRoth** [comment]
@francisagyapong-goa This should be fixed now, if you're able to grab the latest build and see if the problem is resolved for you, that would be great.

**Comment by francisagyapong-goa** [comment]
> @francisagyapong-goa This should be fixed now, if you're able to grab the latest build and see if the problem is resolved for you, that would be great.

@ArakTaiRoth the issue has been resolved in version `1.0.0-alpha.105`

**Comment by ArakTaiRoth** [comment]
Closed as resolved
